ivy-reverse-proxy-nginx-fail2ban/fail2ban/filter.d/nginx-badbots.conf

```diff
@@ -0,0 +1,21 @@
+# Fail2Ban configuration file

```

Comment:
do you really need to add all the fail2ban configuration files. are these not already part of the container itself?

if they are already in the container, then I would not define them here. because they are getting outdated.

Reply:
No, they are not part of this image. I copied them from the swag image. But here I use the fail2ban image itself.

Reply:
I think all this filter.d files are already in the container. Please remove them.